presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/EnvironmentOptions.java

```diff
@@ -24,7 +24,7 @@
     public String hadoopBaseImage = System.getenv().getOrDefault("HADOOP_BASE_IMAGE", "prestodev/hdp2.6-hive");
 
     @Option(name = "--image-version", title = "version", description = "docker images version")
-    public String imagesVersion = System.getenv().getOrDefault("DOCKER_IMAGES_VERSION", "25");
+    public String imagesVersion = System.getenv().getOrDefault("DOCKER_IMAGES_VERSION", "26");
```

Comment:
There is an issue with krb in 26. I fixed that and I need to do another release.